js/packages/teams-ai/src/tokenizers/GPT3Tokenizer.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @module teams-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Tokenizer } from './Tokenizer';
+import { encode, decode } from 'gpt-3-encoder';
+
+/**
+ * Tokenizer that uses GPT-3's encoder.
+ */
+export class GPT3Tokenizer implements Tokenizer {
```

Comment:
would it be possible to use this tokenizer for GPT 4 as well? If so I think renaming to `GPTTokenizer` would make sense.